pyvista/plotting/plotting.py

```diff
@@ -218,6 +219,43 @@ def clear_events_for_key(self, key):
         self._key_press_event_callbacks.pop(key)
 
 
+    def enable_depth_peeling(self, number_of_peels=5, occlusion_ratio=0.1):
+        """Enables depth peeling if supported.
+
+        Parameters
+        ----------
+        number_of_peels: int
+            The maximum number of peeling layers. A value of 0 means no limit.
+
+        occlusion_ratio : float
+            The threshold under which the algorithm stops to iterate over peel
+            layers. A value of 0.0 means that the rendering have to be exact.
+            Greater values may speed-up the rendering with small impact on the
+            quality.
+
+        Returns
+        -------
+        depth_peeling_supported: bool
+            If True, depth peeling is supported.
+        """
+        depth_peeling_supported = check_depth_peeling(number_of_peels,
+                                                      occlusion_ratio)
+        if hasattr(self, 'ren_win') and depth_peeling_supported:
+            self.multi_samples = self.ren_win.GetMultiSamples()
+            self.ren_win.AlphaBitPlanesOn()
+            self.ren_win.SetMultiSamples(0)
```

Comment:
So I'm starting to think that setting the multi-samples to zero isn't needed... or that it might not matter as the multi-samples need to be set before the first render. I deleted this line locally and couldn't tell a difference in the output.

When I added #365, I remember finding some VTK docs about how the multi-samples needs to be set before the first render. At the moment, all I'm digging up is this thread: https://vtk.org/pipermail/vtkusers/2017-November/100211.html

> Make sure you set it before your first render, it is used when creating the
hardware window.

Reply:
It didn't change anything in my local example as well :+1: